src/Enumerables/AlertFrequency.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
+
 namespace EthicalJobs\SDK\Enumerables;
```

Comment:
These really ought to be correctly called "enumerations" rather than "enumerables", but that's probably out of scope for this PR.